sdk/communication/communication-identity/samples-dev/createUserAndToken.ts

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * @summary Create a new user and a token simultaneously.
+ */
+
+import { CommunicationIdentityClient, TokenScope } from "@azure/communication-identity";
+import { CreateUserAndTokenOptions } from "../src";
+
+// Load the .env file if it exists
+import * as dotenv from "dotenv";
+dotenv.config();
+
+// You will need to set this environment variables or edit the following values
+const connectionString =
+  process.env["COMMUNICATION_CONNECTION_STRING"] || "<communication service connection string>";
+
+export async function main() {
+  console.log("\n== Create User and Token Sample ==\n");
+
+  const client = new CommunicationIdentityClient(connectionString);
+  const scopes: TokenScope[] = ["chat"];
+
+  // Create user with default token
+  console.log("Creating User and Token");
+
+  const communicationUserToken = await client.createUserAndToken(scopes);
+
+  console.log(`Created user with id: ${communicationUserToken.user.communicationUserId}`);
+  console.log(`Issued token: ${communicationUserToken.token}`);
+  console.log(`Token expires on: ${communicationUserToken.expiresOn}`);
+
+  // Create user with token with custom expiration
+  console.log("Creating User and Token");
```

Comment:
let's specify it for the console message as well

Reply:
Updated.